File: application/controllers/games.php
<?php
defined('BASEPATH') OR exit('No direct script access allowed');

class games extends CI_Controller {

	public function index()
	{
		$this->load->view('welcome_message');
	}

	public function addgames()
  	{
    	$this->load->model('GamesModel');
    	$name = $this->input->post('name');
    	$genre = $this->input->post('genre');
    	$release_date = $this->input->post('release_date');
    	$provider = $this->input->post('provider');
    	$platform = $this->input->post('platform');
    	$price = $this->input->post('price');
    	$picture ="img/games/";
    	$picture .= $this->input->post('picture');
    	$description = $this->input->post('description');
    	$table = 'games';

    	$data_insert = array (
      	'name' => $name,
      	'genre' => $genre,
      	'release_date' => $release_date,
      	'provider' => $provider,
      	'platform' => $platform,
      	'price' => $price,
      	'picture' => $picture,
      	'description' => $description
    	);

    	$addgames = $this->GamesModel->add_games($table, $data_insert);

    	if ($addgames) {
     		redirect('account/dashboard');
    	}
  	}

	public function delete_game($games_id){
		$this->load->model('GamesModel');
		$delete = $this->GamesModel->delete($games_id);
	
	}

	public function show_game()
	{
		$this->load->model('GamesModel');
		$data = $this->GamesModel->get_all_games();
		$this->load->view('games', $data);
	}

	public function view($games_id)
	{
		$this->load->model('GamesModel');
		$data=$this->GamesModel->get_data($games_id) ;
		$this->load->view('view_games', $data);
	}

	public function view_admin($games_id)
	{
		$this->load->model('GamesModel');
		$data=$this->GamesModel->get_data($games_id) ;
		$this->load->view('admin_view_games', $data);
	}

	public function edit_game($games_id)
	{
		$this->load->model('GamesModel');
		$data=$this->GamesModel->get_data($games_id) ;
		$this->load->view('edit_games', $data);
	}

	public function update_games($games_id){
		$this->load->model('GamesModel');
    	$name = $this->input->post('name');
    	$genre = $this->input->post('genre');
    	$release_date = $this->input->post('release_date');
    	$provider = $this->input->post('provider');
    	$platform = $this->input->post('platform');
    	$price = $this->input->post('price');
    	$picture ="img/games/";
    	$picture .= $this->input->post('picture');
    	$description = $this->input->post('description');
    	$table = 'games';
    	$id = $games_id;

    	$data_update = array (
      	'name' => $name,
      	'genre' => $genre,
      	'release_date' => $release_date,
      	'provider' => $provider,
      	'platform' => $platform,
      	'price' => $price,
      	'picture' => $picture,
      	'description' => $description
    	);

    	$updategames = $this->GamesModel->updategames($table, $data_update, $id);

    	if ($updategames) {
     		redirect('account/dashboard');
    	}
	}

	
}

File: database/games.cs
using System;
using System.Collections;
using System.Collections.Generic;
using System.Text;
namespace Asiaplaystation
{
    #region Games
    public class Games
    {
        #region Member Variables
        protected int _games_id;
        protected string _name;
        protected string _genre;
        protected unknown _release_date;
        protected string _provider;
        protected string _platform;
        protected int _price;
        protected string _picture;
        protected string _description;
        #endregion
        #region Constructors
        public Games() { }
        public Games(string name, string genre, unknown release_date, string provider, string platform, int price, string picture, string description)
        {
            this._name=name;
            this._genre=genre;
            this._release_date=release_date;
            this._provider=provider;
            this._platform=platform;
            this._price=price;
            this._picture=picture;
            this._description=description;
        }
        #endregion
        #region Public Properties
        public virtual int Games_id
        {
            get {return _games_id;}
            set {_games_id=value;}
        }
        public virtual string Name
        {
            get {return _name;}
            set {_name=value;}
        }
        public virtual string Genre
        {
            get {return _genre;}
            set {_genre=value;}
        }
        public virtual unknown Release_date
        {
            get {return _release_date;}
            set {_release_date=value;}
        }
        public virtual string Provider
        {
            get {return _provider;}
            set {_provider=value;}
        }
        public virtual string Platform
        {
            get {return _platform;}
            set {_platform=value;}
        }
        public virtual int Price
        {
            get {return _price;}
            set {_price=value;}
        }
        public virtual string Picture
        {
            get {return _picture;}
            set {_picture=value;}
        }
        public virtual string Description
        {
            get {return _description;}
            set {_description=value;}
        }
        #endregion
    }
    #endregion
}